tiiuae/ghaf
# Audio Control Application

## Description of changes

This PR adds the Ghaf Audio Control Appication to be able control AppVM's audio: speakers, microphones

This application uses Ghaf Audio Control library, which might be integrated in any bar, like Waybar or Ironbar, or into Ghaf Control Panel



## Checklist for things done






## Testing

* Run application in taskbar "Audio Control"
* Check there are some sinks (audio devices)
* Check you can control sound by sliding a slider
